.eslintrc

```diff
@@ -1,12 +1,14 @@
 {
   "env": {
     "node": true,
-    "es6": true
+    "es6": true,
+    "mocha": true
```

Comment:
Added `mocha` option due to getting `described` and `it` are undefined on the CI.